src/routes/login/forgot-password/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { createFileRoute } from "@tanstack/react-router";
+import { useSyncLanguage } from "@/hooks/useSyncLanguage";
+import { ForgotPassword } from "@/components/common/LoginPages/sections/forgot-password-form";
+
+export const Route = createFileRoute("/login/forgot-password/")({
+    component: PasswordReset,
+});
+
+export function PasswordReset() {
+    useSyncLanguage();
```

Comment:
Include globally

Reply:
As discussed, have removed it since we do not need localization for now.